### ISSUE TITLE
NuGet.VisualStudioExtension: Add solution to TFS and then install-packages will automatically add packages folder to TFS

### ISSUE THREAD

**Issue by danliu** [NuGet/Home#184]
Repro step:
1. create a class library project, install some packages
2. Add solution to TFS, which will not add packages folder
3. check in pending changes, install some other packages
4. check pending changes again.

Expected: NuGet should be consistent if Packages folder is considered as the source pending changes

Actual: after step 3, the newly installed packages will be marked with pending changes with their files in Packages folder.

**Comment by feiling** [comment]
Dan thinks this is still not fixed.

**Comment by deepakaravindr** [comment]
This is a duplicate of https://github.com/NuGet/Home/issues/1356
